ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -134,8 +134,9 @@ export class Auth0AdapterService implements OnDestroy {
             }),
             language: this.language.getCurrentLanguage()
         };
-        this.log.logToCloud(`Auth0 login modal is open: ${JSON.stringify(params)}`, { auth0Mode: Auth0Mode.LoginOnly });
-        this.showAuth0Modal(Auth0Mode.LoginOnly, params);
+        this.log.logToCloud(`Auth0 login modal is open: ${JSON.stringify(params)}`, { auth0Mode: Auth0Mode.LoginOnly })
+            .subscribe(() => this.showAuth0Modal(Auth0Mode.LoginOnly, params)
```

Comment:
It is a good catch!  

Just wondering whether using a `webworker` would help us to make the logging calls in a separate tread. 
But please don't go so crazy, it can be just an idea for research time. )

Reply:
Interesting idea! I am going to postpone pursuing that for now, but we might come back to this. Have had the sense that webworker could have a role in our apps and this might be it. But for now I just want to get rid fo the errors.